packages/sui-mono/src/check.js

```diff
@@ -28,6 +26,12 @@ const isCommitBreakingChange = commit => {
   )
 }
 
+const COMMIT_TYPES_WITH_RELEASE = ['fix', 'feat', 'perf', 'refactor']
```

Comment:
Maybe we can move this variable inside the new function: `isCommitReleaseTrigger`